data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -159,7 +160,8 @@
     (valid_death_dates.get_valid_death_date_queries,),
     (fill_source_value.get_fill_freetext_source_value_fields_queries,),
     (repopulate_person.get_repopulate_person_post_deid_queries,),
-    (CleanMappingExtTables,)
+    (CleanMappingExtTables,),
+    (CleanHeightAndWeight,),
```

Comment:
This should be part of `DEID_CLEAN_CLEANING_CLASSES`, not `DEID_BASE_CLEANING_CLASSES`.

Reply:
Also, you should add your rule before the `(CleanMappingExtTables,)` rule.

Reply:
I moved it to `DEID_CLEAN_CLEANING_CLASSES` and put it before `(CleanMappingExtTables,)`

Reply:
Moved 👍🏾